Web.Tests/GetTests.cs

```diff
@@ -17,41 +17,43 @@ public GetTests()
             Initialize();
         }
 
-        #region Fields
         private IServiceProvider serviceProvider;
         private IHttpClientFactory clientFactory;
-        #endregion Fields
 
         private void Initialize()
         {
             IServiceCollection services = new ServiceCollection();
             services.AddHttpClient();
             serviceProvider = services.BuildServiceProvider();
 
-            this.clientFactory = serviceProvider.GetRequiredService<IHttpClientFactory>();
+            clientFactory = serviceProvider.GetRequiredService<IHttpClientFactory>();
```

Comment:
Better not to change this file at all.  Maybe a new file called `EanableApplyWebTests.cs` (depending on what we name the attributes).  Then just copy everything over and update the query e.g. `Test(await Get<CoreBuilding>("/corebuilding/AllowApplyAttribute?$top=5&$expand=Builder($select=Name)&$filter=name eq 'One L1'", port));`

Reply:
If I may I would want to reorganize this file a bit so that it getting a bit more clear structure into it. But you're completely right on that the tests for `EnableQuery` should be in a separate file, so will do 😃

Reply:
Reverted this change. Copied to a new file `AllowApplyTests.cs`